app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,55 @@
+class PostsController < ApplicationController
+  before_action :require_post
+  skip_before_action :require_post, only: [:new, :create]
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    if Post.create(post_params)
+      flash[:notice] = "Post successfully created!"
+      redirect_to new_post_path
+    else
+      flash.now[:alert] = "Error occured while making post!"
+      render :new
+    end
+  end
+
+  def update
+    if @post.update(post_params)
+      flash.now[:notice] = "Post successfully updated!"
+      render :edit
```

Comment:
Principā, parasti pēc veiksmīgas saglabāšanas taisa redirect uz show skatu, lai lietotājs varētu paskatīties ko viņš izdarīja vai redirect uz sarakstu, vai vel kaut kur citur.

Jā tiks izmantots redirect tad instances mainigais @post tiks parlodeta no datubazes un tad būs parliecinātiba kā lietotajs redzes to, kas tagad ir datubaze.

Par to vari palasit šeit: https://en.wikibooks.org/wiki/Ruby_on_Rails/ActionView/Rendering_and_Redirecting

Varbūt kā šeit tev vajadzetu: redirect_to :show, vai :index. Vai jā tev ir svarigi lai tas būtu editešanas skats, tad ari redirect :edit.

Reply:
Nesanaca isti ar `redirect_to :show` ,
bet sanaca ar `redirect_to post_path(@post_id)`